### ISSUE TITLE
Docfx fails generating Metadata for some .dll and .exe files after updating to 2.33

### ISSUE THREAD

**Issue by Herrmel** [dotnet/docfx#2539]
**DocFX Version Used**: 2.33

**Template used**: `default`

**Steps to Reproduce**: Can't give a problemspecific examplecode yet

**Expected Behavior**: Extracts Metadata like it does in 2.32

**Actual Behavior**: stops with an Error saying: "Error extracting metadata for {list of all dll and exe files specified in metadata}: Object reference not set to an instance of an object"

I can track down the specific files causing the error with the last output of "Loading developer comments from file:{filepath}". It's simply the next file in the folder. I just cant find the specific properties of these files. Any Idea what the problem may be or what I should look for?

My current Workaround for this is to generate Metadata with v2.32.2 and Build with v2.33

**Comment by vicancy** [comment]
Are you using Merge command? Could you provide a minimum repro step so I can look into the issue?

**Comment by Herrmel** [comment]
No I dont use merge. Thats the problem...since im not allowed to give you the full file I will have to find the reason thats causing the error in these files but I dont know what to look for.

**Comment by vicancy** [comment]
Hi Herrmel, you can `git clone` our repo and do a local debug, no preparation steps, open `All.sln` is enough to build I believe.

**Comment by Herrmel** [comment]
Ok did that but doesnt work with that version as well and got some unhandled Exceptions where I only got warnings before. I also tested 2.33.1 same result as with 2.33. I'll try to ask someone responsible for the specific files if they have any idea but it might take some time since everyone is on vacation currently. If you have any other ideas let me know.

**Comment by Herrmel** [comment]
Ok tried debug on another machine now and works there. I get a NullReferenceException in "Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn.dll". After further inspection I found out that in GetLiteralString() in RoslynFilterData.cs the var value is null and then it tries to call toString(). when I change value.ToString() to value?.ToString() it goes through but I get a lot of Popups like this
![image](https://user-images.githubusercontent.com/36636591/37960170-35f73e28-31b5-11e8-9d63-122fa8968570.png)
***
Here's the Stacktrace of the nullpointerexception:
**bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynFilterData.GetLiteralString(TypedConstant constant) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Filters\RoslynFilterData.cs:Zeile 105.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynFilterData.<>c.<GetAttributeFilterData>b__1_1(KeyValuePair`2 pair) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Filters\RoslynFilterData.cs:Zeile 29.
   bei System.Linq.ImmutableArrayExtensions.ToDictionary[TKey,TElement,T](ImmutableArray`1 immutableArray, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   bei System.Linq.ImmutableArrayExtensions.ToDictionary[TKey,TElement,T](ImmutableArray`1 immutableArray, Func`2 keySelector, Func`2 elementSelector)
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynFilterData.GetAttributeFilterData(AttributeData attribute) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Filters\RoslynFilterData.cs:Zeile 25.
   bei System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   bei Microsoft.DocAsCode.Metadata.ManagedReference.AttributeFilterInfo.ContainedIn(SymbolFilterData symbol) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Common\Filters\AttributeFilterInfo.cs:Zeile 28.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ConfigFilterRuleItem.IsMatch(SymbolFilterData symbol) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Common\Filters\ConfigFilterRuleItem.cs:Zeile 45.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ConfigFilterRule.CanVisitCore(IEnumerable`1 ruleItems, SymbolFilterData symbol) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Common\Filters\ConfigFilterRule.cs:Zeile 40.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ConfigFilterRule.CanVisitApi(SymbolFilterData symbol) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Common\Filters\ConfigFilterRule.cs:Zeile 27.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ConfigFilterVisitor.CanVisitApiCore(ISymbol symbol, Boolean wantProtectedMember, IFilterVisitor outer) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\ConfigFilterVisitor.cs:Zeile 41.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.DelegatingFilterVisitor.CanVisitApi(ISymbol symbol, Boolean wantProtectedMember, IFilterVisitor outer) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\DelegatingFilterVisitor.cs:Zeile 19.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.CachedFilterVisitor.CanVisitApiCore(ISymbol symbol, Boolean wantProtectedMember, IFilterVisitor outer) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\CachedFilterVisitor.cs:Zeile 29.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.DelegatingFilterVisitor.CanVisitApi(ISymbol symbol, Boolean wantProtectedMember, IFilterVisitor outer) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\DelegatingFilterVisitor.cs:Zeile 19.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.<>c__DisplayClass37_0.<AddInheritedMembers>b__1(ISymbol m) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 730.
   bei System.Linq.Enumerable.<>c__DisplayClass6_0`1.<CombinePredicates>b__0(TSource x)
   bei System.Linq.Enumerable.WhereArrayIterator`1.MoveNext()
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.AddInheritedMembers(INamedTypeSymbol symbol, INamedTypeSymbol type, Dictionary`2 dict, IReadOnlyList`1 typeParamterNames) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 728.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.GenerateInheritance(INamedTypeSymbol symbol, MetadataItem item) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 614.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitNamedType(INamedTypeSymbol symbol) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 179.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitDescendants[T](IEnumerable`1 children, Func`2 getChildren, Func`2 filter) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 559.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitNamespace(INamespaceSymbol symbol) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 163.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitDescendants[T](IEnumerable`1 children, Func`2 getChildren, Func`2 filter) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 559.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitAssembly(IAssemblySymbol symbol) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 147.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynMetadataExtractor.Extract(ExtractMetadataOptions options) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\ExtractMetadata\RoslynMetadataExtractor.cs:Zeile 47.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynIntermediateMetadataExtractor.GenerateYamlMetadata(Compilation compilation, IAssemblySymbol assembly, ExtractMetadataOptions options) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\ExtractMetadata\RoslynIntermediateMetadataExtractor.cs:Zeile 36.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynIntermediateMetadataExtractor.Extract(IInputParameters key) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\ExtractMetadata\RoslynIntermediateMetadataExtractor.cs:Zeile 24.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynSourceFileBuildController.ExtractMetadata(IInputParameters parameters) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\ExtractMetadata\BuildControllers\RoslynSourceFileBuildController.cs:Zeile 21.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ExtractMetadataWorker.GetMetadataFromProjectLevelCache(IBuildController controller, IInputParameters key) in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference\ExtractMetadataWorker.cs:Zeile 487.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ExtractMetadataWorker.<SaveAllMembersFromCacheAsync>d__12.MoveNext() in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference\ExtractMetadataWorker.cs:Zeile 356.
--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
   bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   bei System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ExtractMetadataWorker.<ExtractMetadataAsync>d__10.MoveNext() in D:\docfx-2.33\docfx-2.33\src\Microsoft.DocAsCode.Metadata.ManagedReference\ExtractMetadataWorker.cs:Zeile 103.**

**Comment by vicancy** [comment]
Looks like the `pair.Value` is null in `GetAttributeFilterData`
![image](https://user-images.githubusercontent.com/668244/38003324-b31560ba-3269-11e8-9ac4-0e9fe54d3bc0.png)

Could you share the value of `attribute.NameArguments` for `GetAttributeFilterData`?
This method is called by:
![image](https://user-images.githubusercontent.com/668244/38003406-fdadc4e6-3269-11e8-9c19-388ea2231277.png)
![image](https://user-images.githubusercontent.com/668244/38003412-02f58c04-326a-11e8-8b78-612a06590422.png)
Could you share the value of `symbol.ToDisplayString` when exception happens? 

BTW, are you using c# 7.2 language features?

**Comment by Herrmel** [comment]
`attribute.NameArguments` for each case the Error ocurrs:
```
[0] = {[IsRequired, Microsoft.CodeAnalysis.TypedConstant]}
[1] = {[DefaultValue, Microsoft.CodeAnalysis.TypedConstant]}

IsRequired.value = false
IsRequired.isNull = false
IsRequired.Type = {bool}

DefaultValue.value = null
DefaultValue.isNull = true
DefaultValue.Type = {string}
```
`symbol.ToDisplayString`: 
`"Fis.Dekoder.Configuration.ConsoleElement.WriteExtendedData"`
and 
`"Hermos.Utilities.SpecialFoldersSection.DefaultVersionIsolationLevel" `

Hope I got all relevant Data from `NamedArguments`. Some of our files are using C#7.2 but not the files where I'm getting the error's.

**Comment by vwxyzh** [comment]
Is it same issue with #2547?

**Comment by Herrmel** [comment]
I dont think so. I dont get `The given key was not present in the dictionary` in the Error message, I get `Object reference not set to an instance of an object`. It calls `ToString()` on a Null value. But the Error occurrs in the same Method though.

**Comment by vicancy** [comment]
Repro-ed with:
```cs
using System;


namespace ConsoleApp4
{
    public class Program
    {
        [EnumDisplay(Test = null)]
        public void Test()
        {

        }

        public class EnumDisplayAttribute : Attribute
        {
            public string Test { get; set; }

            public EnumDisplayAttribute()
            {
            }
        }
    }
}
```

**Comment by Herrmel** [comment]
Hello again @vicancy 
I got a new NullReference Exception after Updating to 2.35.4 or 2.35 so I debugged again. In RoslynFilterData.cs in Line 110 the `constant.Values` is `Uninitialized`.
<details> 
<summary>Stacktrace</summary>
   bei System.Linq.ImmutableArrayExtensions.Select[T,TResult](ImmutableArray`1 immutableArray, Func`2 selector)
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynFilterData.GetLiteralString(TypedConstant constant) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Filters\RoslynFilterData.cs:Zeile 110.
   bei System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   bei System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   bei System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynFilterData.GetAttributeFilterData(AttributeData attribute) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Filters\RoslynFilterData.cs:Zeile 25.
   bei System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   bei Microsoft.DocAsCode.Metadata.ManagedReference.AttributeFilterInfo.ContainedIn(SymbolFilterData symbol) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Common\Filters\AttributeFilterInfo.cs:Zeile 28.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ConfigFilterRuleItem.IsMatch(SymbolFilterData symbol) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Common\Filters\ConfigFilterRuleItem.cs:Zeile 45.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ConfigFilterRule.CanVisitCore(IEnumerable`1 ruleItems, SymbolFilterData symbol) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Common\Filters\ConfigFilterRule.cs:Zeile 40.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ConfigFilterRule.CanVisitApi(SymbolFilterData symbol) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Common\Filters\ConfigFilterRule.cs:Zeile 27.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ConfigFilterVisitor.CanVisitApiCore(ISymbol symbol, Boolean wantProtectedMember, IFilterVisitor outer) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\ConfigFilterVisitor.cs:Zeile 41.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.DelegatingFilterVisitor.CanVisitApi(ISymbol symbol, Boolean wantProtectedMember, IFilterVisitor outer) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\DelegatingFilterVisitor.cs:Zeile 19.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.CachedFilterVisitor.CanVisitApiCore(ISymbol symbol, Boolean wantProtectedMember, IFilterVisitor outer) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\CachedFilterVisitor.cs:Zeile 29.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.DelegatingFilterVisitor.CanVisitApi(ISymbol symbol, Boolean wantProtectedMember, IFilterVisitor outer) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\DelegatingFilterVisitor.cs:Zeile 19.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.<>c__DisplayClass37_0.<AddInheritedMembers>b__1(ISymbol m) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 730.
   bei System.Linq.Enumerable.<>c__DisplayClass6_0`1.<CombinePredicates>b__0(TSource x)
   bei System.Linq.Enumerable.WhereArrayIterator`1.MoveNext()
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.AddInheritedMembers(INamedTypeSymbol symbol, INamedTypeSymbol type, Dictionary`2 dict, IReadOnlyList`1 typeParamterNames) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 728.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.GenerateInheritance(INamedTypeSymbol symbol, MetadataItem item) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 614.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitNamedType(INamedTypeSymbol symbol) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 179.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitDescendants[T](IEnumerable`1 children, Func`2 getChildren, Func`2 filter) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 559.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitNamespace(INamespaceSymbol symbol) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 163.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitDescendants[T](IEnumerable`1 children, Func`2 getChildren, Func`2 filter) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 559.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.SymbolVisitorAdapter.VisitAssembly(IAssemblySymbol symbol) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\Visitors\SymbolVisitorAdapter.cs:Zeile 147.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynMetadataExtractor.Extract(ExtractMetadataOptions options) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\ExtractMetadata\RoslynMetadataExtractor.cs:Zeile 47.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynIntermediateMetadataExtractor.GenerateYamlMetadata(Compilation compilation, IAssemblySymbol assembly, ExtractMetadataOptions options) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\ExtractMetadata\RoslynIntermediateMetadataExtractor.cs:Zeile 36.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynIntermediateMetadataExtractor.Extract(IInputParameters key) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\ExtractMetadata\RoslynIntermediateMetadataExtractor.cs:Zeile 24.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.RoslynSourceFileBuildController.ExtractMetadata(IInputParameters parameters) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn\ExtractMetadata\BuildControllers\RoslynSourceFileBuildController.cs:Zeile 21.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ExtractMetadataWorker.GetMetadataFromProjectLevelCache(IBuildController controller, IInputParameters key) in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference\ExtractMetadataWorker.cs:Zeile 487.
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ExtractMetadataWorker.<SaveAllMembersFromCacheAsync>d__12.MoveNext() in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference\ExtractMetadataWorker.cs:Zeile 356.
--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
   bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   bei System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   bei Microsoft.DocAsCode.Metadata.ManagedReference.ExtractMetadataWorker.<ExtractMetadataAsync>d__10.MoveNext() in D:\docfx-2.35.4\src\Microsoft.DocAsCode.Metadata.ManagedReference\ExtractMetadataWorker.cs:Zeile 103.
</details>

Thanks for your support
